pkg/core/native/native_neo.go

```diff
@@ -26,7 +26,8 @@ type NEO struct {
 	nep5TokenNative
 	GAS *GAS
 
-	validators atomic.Value
+	nextValidators atomic.Value
```

Comment:
I don't like this name, because this cache is not like `validators`, however `lastNextValidators` is more misleading imo.
Suggestions are welcome :)